<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>count</class>
 <widget class="QDialog" name="count">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>选择排序方式</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按照价格统计</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按照库存统计</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按照生产厂家统计</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
